core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -5383,6 +5383,8 @@ groups:
     - openshift-release-infra-periodics.yaml
     - infra-periodics-origin-release-images.yaml
   build02:
+    jobs:
+    - oc-mirror-e2e-unit
```

Comment:
Could you please name the job as `org-repo-branch-testname`?

Reply:
Actually, can you do this together with https://github.com/openshift/release/pull/24134/files ?

Reply:
Sure! I'll do this today. Thanks!

Reply:
I have combined the two into https://github.com/openshift/release/pull/24492